src/python/grpcio_tests/tests/interop/methods.py

```diff
@@ -291,36 +315,89 @@ def _empty_stream(stub):
 
 
 def _status_code_and_message(stub):
-  message = 'test status message'
-  code = 2
-  status = grpc.StatusCode.UNKNOWN # code = 2
+  # Function wide constants
+  _MESSAGE = 'test status message'
```

Comment:
Alternatively, you could pull them out of the behavior and make them module-scope constants. Locality to their use-sites isn't the only consideration.